docs/en/stack/ml/df-analytics/dfanalytics-limitations.asciidoc

```diff
@@ -104,6 +104,21 @@ where included fields contain an array are also ignored. Documents in the
 destination index that don't contain a results field are not included in the 
 {classanalysis}.
 
+[float]
+[[dfa-classification-imbalanced-classes]]
+=== Imbalanced class sizes affect {classification} performance
+
+If your training data is very imbalanced, then {classanalysis} may not provide 
+good predictions. The system tries to mitigate the effects of imbalanced 
+training data by maximizing the minimum recall of any class. It means that it 
+tries to get correct the highest number of examples in the minority classes. 
+However, this process can result in a slight degradation of the overall 
+accuracy. Try to avoid highly imbalanced situations. It is required at least 50 
```

Comment:
I'd weaken these words slightly (and suggest a maximum imbalance), i.e. something like:
"We recommend having at least 50 examples of each class and a ratio of no more than 10 to 1 for the majority to minority class labels in the training data."